resistance-is-futile/usr/share/asteroid-launcher/watchfaces/resistance-is-futile.qml

```diff
@@ -0,0 +1,157 @@
+        color: "transparent"
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: -parent.height * 0.272
+        }
+        width: parent.width * 0.12
+        height: parent.height * 0.12
+
+
+        Image {
+            z: 1
+            id: asteroidLogo
+            visible: !displayAmbient
+            source: "../watchfaces-img/asteroid-logo.svg"
+            antialiasing: true
+            anchors.fill: parent
+            opacity: 0.7
+
+            Text {
+                z: 2
+                id: asteroidSlogan
+                visible: !displayAmbient
+                font {
+                    family: "Raleway"
+                    pixelSize: parent.height * 0.31
+                }
+                color: "white"
+                horizontalAlignment: Text.AlignHCenter
+                anchors {
+                    centerIn: parent
+                    verticalCenterOffset: -parent.height * 0.005
+                }
+                text: "<b>AsteroidOS</b><br>Hack Your Wrist"
+            }
+        }
+        MouseArea {
+            anchors.fill: parent
+            onPressAndHold: asteroidLogo.visible ?
+                                asteroidLogo.visible = false :
+                                asteroidLogo.visible = true
+        }
+    }
+
+    Text {
+        id: datetext
+        anchors {
+            centerIn: parent
+            verticalCenterOffset: parent.height * 0.25 
+        }
+        font {
+            family: "Titillium"
+            weight: Font.Thin
+            pixelSize: parent.height * 0.06
+        }
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ddd dd");
+    }
+
+    Rectangle {
+        id: time
+        color: displayAmbient ? "#371102" : "#6e2304"
+        anchors.centerIn: parent
+        height: parent.height * 0.30
+        width: parent.width
+        Repeater{
+            id: digits
+            model: 4
+            Rectangle {
+                color: ["black","brown","red","orange","yellow","green","blue","violet","gray","white"][getTimeDigit(wallClock.time, index)]
+                opacity: displayAmbient ? 0.7 : 1
+                height: parent.height
+                width: parent.width * 0.1
+                x: parent.width / 5 * index + parent.width / 10
+            }
+        }
+    }
+
+    Text {
+        id: ampm
+        visible: use12H.value
+        anchors {
+            centerIn: parent
+            horizontalCenterOffset: parent.width * 0.4
+        }
+        font {
+            family: "Titillium"
+            weight: Font.Bold
+            pixelSize: parent.height * 0.05
+        }
+        color: "white"
+        text: wallClock.time.toLocaleString(Qt.locale(), "ap");
+    }
+
+    Repeater{
+        id: secondMarks
+        model: 60
+        Rectangle {
+            id: second
+            visible: !displayAmbient
+            antialiasing : true
+            color: "transparent"
```

Comment:
Rather than a transparent Rectangle you should be able to use an Item right ?

Reply:
Or just merge the children Rectangle into this one, I don't think this would have consequences ?

Reply:
Even better -- I changed to use a resistor symbol (American style -- sorry, I thought it looked better!)